lib/compiler/build_runner.zig

```diff
@@ -431,6 +431,14 @@ pub fn main() !void {
                 .windows => fatal("--fuzz not yet implemented for {s}", .{@tagName(builtin.os.tag)}),
                 else => {},
             }
+            switch (@bitSizeOf(usize)) {
+                // Current implementation depends usize being 64bits,
+                // More specifically, posix.mmap()'s second parameter, `length: usize`,
+                // must be the same bit-size as std.fs.getEndPos()'s return value, which is `u64`.
+                // Affects or affected by issues #5185, #22523, and #22464.
+                64 => {},
```

Comment:
This allows fuzz testing only on 64-bit platforms, but it's not enough to fix the build runner on 32-bit platforms.